nightingale_interface/screens/confirmation_screen.py

```diff
@@ -0,0 +1,115 @@
+from kivy.uix.screenmanager import Screen
+from kivy.uix.image import Image
+
+from kivymd.uix.label import MDLabel
+from kivymd.uix.button import MDRectangleFlatButton
+from kivy.uix.button import Button
+
+from kivy.uix.screenmanager import SlideTransition, NoTransition
+from screens.screen_config import ScreenConfig as cfg
+
+
+class ConfirmationScreen:
+    def estop(self, button_data):
+        button_data.parent.manager.transition = NoTransition()
+        cfg.LAST_SCREEN = "confirmationscreen"
+        button_data.parent.manager.current = "estoppedscreen"
+
+    def reset_counts(self):
+        # only reset if starting new selection
+        if cfg.LAST_SCREEN == "itemfillscreen":
+            self.water_count = 0
+            self.ice_count = 0
+            self.blanket_count = 0
+
+    def press(self, button_data):
+        if button_data.id == "yes":
+            # do yes and return home
+            button_data.parent.manager.transition = NoTransition()
+            cfg.LAST_SCREEN = "confirmationscreen"
+
+            def execute_action():
+                # state machine to do things based on the executed action
+                # if given a 'yes' confirmation
+                cfg.CURRENT_ACTION = cfg.PENDING_ACTION
+                cfg.PENDING_ACTION = ""
+                print(f"CUR ACTION {cfg.CURRENT_ACTION}")
+
+                # reset counters regardless of cancel or send
+                self.reset_counts()
+
+                if cfg.CURRENT_ACTION == cfg.NO_ROS_ACTION:
```

Comment:
Will be doing some more design/thinking about what actions are needed and how the topics should be structured